src/Games/NexusMods.Games.StardewValley/Emitters/SMAPIGameVersionDiagnosticEmitter.cs

```diff
@@ -40,7 +40,7 @@ public async IAsyncEnumerable<Diagnostic> Diagnose(Loadout.ReadOnly loadout, [En
         if (gameToSMAPIMappings is null) yield break;
 
         // var gameVersion = SimplifyVersion(new Version("1.5.6.22018"));
-        var gameVersion = new SemanticVersion(loadout.InstallationInstance.Version);
+        var gameVersion = new SemanticVersion(loadout.GameVersion);
```

Comment:
For the Stardew Valley diagnostic emitters, we should use the assembly file version of the `Stardew Valley.dll` file.

Reply:
we can add that in, in a future PR, this keeps the existing behavior